### PR TITLE
English version of README (and improvements of contents)

### PR DESCRIPTION
**Pull request by ytorii**
### Overview:概要
Created README_EN.md and improved contents 

### Impact point:変更に関する影響箇所
- Created README_EN.md
Based on [plan 1 of issue #113](https://github.com/shinosakarb/tebukuro/issues/113),  edited English edition of README as README_EN.md.

- Improved contents
In README_EN.md, improved 'What is Tebukuro?' and 'Features ' to emphasize the concepts of Tebukuro , based on [the issue #19](https://github.com/shinosakarb/tebukuro/issues/19). If these changes are OK, I'd like to reflect those to japanese README.md.

Please also check  [README_EN.md here](https://github.com/ytorii/tebukuro/blob/README_EN_r0.0/README_EN.md).

### CODE REVIEW

**Comment by gazayas** [comment]
I'm not home right now, but I'll check it soon

**Comment by ytorii** [comment]
Thank you, please review when you have time.

**Comment by gazayas** [comment]
厳し目にReviewをさせていただきました 😅 

I think you did a good job though. Thank you for writing the readme

File: README_EN.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Tebukuro is the 'Mobile-friendly' open-source web service, enabling you to manage everything about your communities and events only from mobile devices. Furthermore, supporting communication while holding events and after closed events, not only annoucements and receptions before events.
+
+## Features
+
+- **Mobile-friendly**
+  - [ ] Managing communities and events from mobile devices.
+
+- **Energizing your communities**
+  - [ ] Simple procedure for releasing new events to communities' site.
```
Comment (ytorii) [inline README_EN.md]: I meant that our users can easily add application forms for events in their community's page, inside Tebukuro. @murajun1978, is that correct understanding? 

> Tebukuroのコンセプト
>- モバイルフレンドリー
>- モバイルのみで簡単にコミュニティやイベントを公開できる
>- イベント中の写真をアップロード（SNS通知）
>- コミュニティサイトに申し込みページを簡単に追加できる
>- プライベートなイベントを公開できる

**Comment by ytorii** [comment]
Thanks for checking my poor English :joy:
I'd like to correct the passages you have pointed out!

File: README.md [hunk 2]
```diff
@@ -3,8 +3,11 @@
 
 オープンソースなイベント運営管理サービス
 
+English README is available at README_EN.md!
```
Comment (ytorii) [inline README.md]: Added "The" at the beginning.

File: README_EN.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Tebukuro
+[![wercker status](https://app.wercker.com/status/35e08da3b0a742e5c50ffac2670a821b/s/master "wercker status")](https://app.wercker.com/project/byKey/35e08da3b0a742e5c50ffac2670a821b)
+
+A 'Mobile-friendly' open-source web service for events and communities management
```
Comment (ytorii) [inline README_EN.md]: Changed sentences as you suggested :smiley:

File: README_EN.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+<img src="http://4.bp.blogspot.com/-dMHKByNfZxo/Uku9_iuHWHI/AAAAAAAAYqg/LjlcTLrN44o/s800/winter_tebukuro.png" width="320px">
+
+## What is tebukuro?
+
+Tebukuro is the 'Mobile-friendly' open-source web service, enabling you to manage everything about your communities and events only from mobile devices. Furthermore, supporting communication while holding events and after closed events, not only annoucements and receptions before events.
```
Comment (ytorii) [inline README_EN.md]: Yes, that's what I wanted to say!
> "Not only ..., but ... as well."
> I think this is what you're looking for.

File: README_EN.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Features
+
+- **Mobile-friendly**
+  - [ ] Managing communities and events from mobile devices.
```
Comment (ytorii) [inline README_EN.md]: Removed -ing from the list of features.

File: README_EN.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+- **Energizing your communities**
```
Comment (ytorii) [inline README_EN.md]: Removed -ing from the list of features as well.

File: README_EN.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## Requirement
+
+- Ruby on Rails
+- React
+- Redux
+
+## Contributing
+
+Please feel free to fork and join us! This project is also a place for improving members' skill and testing the latest technologies.
```
Comment (ytorii) [inline README_EN.md]: Added "s" to skills.

File: README_EN.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+# Tebukuro
+[![wercker status](https://app.wercker.com/status/35e08da3b0a742e5c50ffac2670a821b/s/master "wercker status")](https://app.wercker.com/project/byKey/35e08da3b0a742e5c50ffac2670a821b)
+
+ "Mobile Friendly" open-source web service for managing events and communities.
```
Comment (gazayas) [inline README_EN.md]: Don't forget "a" at the beginning!

File: README_EN.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+Tebukuro is a "Mobile Friendly" open-source web service, enabling you to manage your communities and events solely from mobile devices. Furthermore, not only is there support for announcements and reception before events, but for communication during events and ones that have already closed as well.
+
+## Features
+
+- **Mobile Friendly**
+  - [ ] Manage communities and events from mobile devices
+
+- **Energize your communities**
+  - [ ] Simple procedure for releasing new events to communities' site
```
Comment (gazayas) [inline README_EN.md]: I see, so it's the community's page.

I suggest writing this:
```
Simple procedure for opening new events in communities.
```

I think this is a simple way of saying what you want to say.

**Comment by gazayas** [comment]
この２つだけです (-o-)V

File: README_EN.md (hunk 8 above)
Comment (ytorii) [inline README_EN.md]: Oops!

File: README_EN.md (hunk 9 above)
Comment (ytorii) [inline README_EN.md]: Thank you, it's very simple!

**Comment by ytorii** [comment]
Thank you for re-review, I would fix those passages and commit.

**Comment by ytorii** [comment]
Corrected README_EN.md with @gazayas and reflected changes in README_EN.md to README.md. So if there is no problem, please merge this pull request.

**Pull request merged**